neuralmonkey/learning_utils.py

```diff
@@ -73,11 +73,10 @@ def training_loop(tf_manager: TensorFlowManager,
             "Logging period can't smaller than validation period.")
     _check_series_collisions(runners, postprocess)
 
-    paramstr = "Model has {} trainable parameters".format(trainer.n_parameters)
-    if tf_manager.report_gpu_memory_consumption:
-        paramstr += ", GPU memory usage: {}".format(gpu_memusage())
+    _log_model_variables()
 
-    log(paramstr)
+    if tf_manager.report_gpu_memory_consumption:
```

Comment:
Těžko říct. Ale asi spíš ne. Když TF alokuje najednou tak sní rovnou všechno a když postupně, tak to není zajímavý.